backend/index.js

```diff
@@ -0,0 +1,35 @@
+import fs from "fs";
+import express from "express";
+
+const app = express();
+
+let sessionCounter = 0;
+let counter;
+
+try {
+  fs.existsSync("temp/my-counter-visited-times.txt")
+    ? (counter = fs.readFileSync("temp/my-counter-visited-times.txt", "utf8"))
+    : (counter = 0);
+} catch (e) {
+  console.log(e);
+}
+
+app.get("/", (req, res) => {
+  sessionCounter++;
+  counter = parseInt(counter);
+  res.send(
+    `This page was visited ${
+      counter + sessionCounter
+    } times in total, ${sessionCounter} out of which was during the current code execution session ( statuscode:${
+      res.statusCode
+    } ) `
+  );
+
+  let str = JSON.stringify(counter + sessionCounter);
```

Comment:
It's not modified so you can use const instead.

Reply:
Also because it's not a JSON at this point you can just do `String(counter + sessionCounter)`

Reply:
`str` is not a meaningful name. Please think of a better name, or you can just use it right away e.g. fs.writeFile(..., String(....))